crates/swc_ecma_minifier/src/metadata/mod.rs

```diff
@@ -11,6 +14,37 @@ use swc_ecma_visit::{
 
 use crate::option::CompressOptions;
 
+#[derive(Debug, Eq)]
+struct HashEqIgnoreSpanExprRef<'a>(&'a Expr);
```

Comment:
I'm not sure if this will work.

Reply:
Can you add some tests?

Reply:
Sure. I guess we only need to test the `Eq` and `Hash` behaviors of `HashEqIgnoreSpanExprRef<'a>`.

Reply:
I think inserting and getting various values from `HashSet<HashEqIgnoreSpanExprRef>` is enough